internal/test/wmcb/wmcb_test.go

```diff
@@ -58,9 +55,9 @@ var (
 		Value:  "Windows",
 		Effect: v1.TaintEffectNoSchedule,
 	}
-	// filesToBeTransferred holds the list of files that needs to be transferred to the Windows VM
-	filesToBeTransferred = flag.String("filesToBeTransferred", "",
-		"Comma separated list of files to be transferred")
+	// dirsToBeTransferred holds the list of folders that needs to be transferred to the Windows VM
```

Comment:
directories

Reply:
Not sure what you mean by saying `directories`.

Please replace `folders` with `directories`.